bears/python/BlackBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class BlackBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    REQUIREMENTS = {PipRequirement('black', '18.3a2')}
```

Comment:
we could add a `DistributionRequirement('python3', version='3.6')` and then our build parsers could be enhanced to detect that and skip the bear tests on lower versions.

A quick hack is to edit the build scripts to delete the bear & its tests when the job is < 3.6